felix/vxlanfdb/vxlan_fdb.go

```diff
@@ -77,12 +77,13 @@ type VXLANFDB struct {
 	ifIndex        int
 	arpEntries     *deltatracker.DeltaTracker[ip.Addr, comparableHWAddr]
 	fdbEntries     *deltatracker.DeltaTracker[comparableHWAddr, ip.Addr]
-	logCxt         *log.Entry
+	logCtx         *log.Entry
 	resyncPending  bool
 	logNextSuccess bool
 	nl             *handlemgr.HandleManager
 
 	newNetlinkHandle func() (netlinkshim.Interface, error)
+	arpOnly          bool
```

Comment:
Having a bool that turns off half the function isn't ideal.  Could we split this struct into two, one for ARP and one for the FDB?  (And then only instantiate one of them?)

Or, _should_ we be resyncing ARP too but cleaning it out if we're in the mode that doesn't use it?  In which case you could use the bool to prevent population of the Desired() side of the arpEntries tracker and it'd naturally clean everything up.

Reply:
It is the opposite, we always need arp/neigh, we do not need fdb in bpf mode :eyes:

Reply:
will merge as is and created a ticket for this